### ISSUE TITLE
Simulation features

### ISSUE THREAD

**Issue by esheldon** [LSSTDESC/descwl-shear-sims#41]
- [ ] realistic offsets for images
- [ ] bright star masks
- [ ] bleed trail masks
- [ ] different psf models
- [ ] spatial psf variation
- [ ] Correlated noise from brighter-fatter correction
- [ ] distortion terms in wcs
- [ ] edge distortions (not yet supported by DM WCS)
- [ ] tree rings (not yet supported by DM WCS)
- [ ] chromatic psf effects (not yet supported by DM PSF models)

**Comment by esheldon** [comment]
For "realistic offsets for images" I was imagining that we simulate 4k x 4k images with offsets that are large but for which the coadd region is still contained within it

**Comment by esheldon** [comment]
This way our TanWCS has a center that is farther from the coadd position

**Comment by beckermr** [comment]
Right. For each bulletin above we ahould make a separate issue with details like this when we think of them.
